index.html

```diff
@@ -7,92 +7,226 @@
-        <div class="flex_container"><a href="#">book</a><a href="#">explore</a><a href="#">about</a></div>
-        <div><a href="#">sign in</a></div>
-      </nav>
+
+  <div class="page_wrapper">
+
+    <header class="header">
+      <div class="header_wrapper">
+        <div class="section_wrapper section header flex_container">
+          <h2 class="logo">
+            ReMotely
+          </h2>
+
+          <nav class="nav flex_container">
+            <ul class="link_flex_container">
+              <li><a class="links" href="#">book</a></li>
+              <li><a class="links" href="#">explore</a></li>
+              <li><a class="links" href="#">about</a></li>
+            </ul>
+            <div class="sign_container">
+              <a class="sign" href="#">sign in</a>
+            </div>
+          </nav>
+        </div>
+      </div>
     </header>
 
 
-    <div class="catch_phrase">
-      <h1>Work from a Cabin near Helsinki</h1>
-      <button class="btn" type="submit">Explore</button>
-      <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
-    </div>
-  </div>
+    <section class="green_section">
+      <div class=" section_hero">
+        <div class="section_container">
+          <div class="column phrase">
+            <div class="catch_phrase">
+              <h1 class="invite_phrase">
+                Work from a&nbsp;cabin near Helsinki
+              </h1>
+              <div class=button_container>
+                <button class="btn" type="submit">Explore</button>
+              </div>
+            </div>
+          </div>
+          <div class="column cottage">
+            <div class="photo">
+              <img class="main_photo" src="./switzphoto.jpg" alt="Swiss Cottage">
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
 
 
-  <div class="second_page_container">
-    <div class="paragraph">
-      <p>We offer a handpicked range of cabins for nature lovers in Europeto live and work from for desired time.</p>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Connection</header>
-          <p>We provide a good internet connection in our remotest cabins</p>
+    <section class="section second_page_container">
+      <h2 class="section_heading">
+        We offer a handpicked range of cabins for nature lovers in Europe to live and work
+        from for desired time.
+      </h2>
+      <div class="all_pros_container">
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo">
+              <img class="pros_pic" src="./forest.png" alt="pic of pros">
+            </div>
+            <div class="header_container">
+              <h3 class="paragraph_header">Connection</h3>
```

Comment:
those headings should have some margin, check in figma

Reply:
fixed